src/main/java/mil/dds/anet/database/LocationDao.java

```diff
@@ -60,6 +62,31 @@ public int updateInternal(Location l) {
     return AnetObjectEngine.getInstance().getSearcher().getLocationSearcher().runSearch(query);
   }
 
+  @InTransaction
+  public int mergeLocations(Location loserLocation, Location winnerLocation) {
+    final String loserLocationUuid = loserLocation.getUuid();
+    final String winnerLocationUuid = winnerLocation.getUuid();
+
+    // Update Locations
+    update(winnerLocation);
+
+    // Update approvalSteps
+    updateForMerge("approvalSteps", "relatedObjectUuid", winnerLocationUuid, loserLocationUuid);
+
+    // Update reports location
+    updateForMerge("reports", "locationUuid", winnerLocationUuid, loserLocationUuid);
+
+    // update positions location
+    updateForMerge("positions", "locationUuid", winnerLocationUuid, loserLocationUuid);
+
+    // update noteRelatedObjects location
+    updateM2mForMerge("noteRelatedObjects", "noteUuid", "relatedObjectUuid", winnerLocationUuid,
+        loserLocationUuid);
+
+    // finally delete the location!
```

Comment:
Consistently use initial cap in comments, i.e. "Update …" / "Finally, delete …".